src/zope/applicationcontrol/tests/test_runtimeinfo.py

```diff
@@ -119,14 +142,56 @@ def test_Uptime(self):
 
         # get the uptime the current implementation calculates
         test_uptime = runtime_info.getUptime()
+        self.assertAlmostEqual(asserted_uptime, test_uptime, delta=time_tolerance)
 
-        self.assertTrue(abs(asserted_uptime - test_uptime) < time_tolerance)
+    @unittest.skipIf(not sys.path, "Need entries on path")
```

Comment:
I wouldn't think so under normal circumstances, but if it did that test would be providing false assurance. I suppose an alternative would have been an `assertTrue(sys.path)` in the body of the test.